modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Manage/User/UserForm.tsx

```diff
@@ -154,6 +161,9 @@ const UserForm = () => {
 		register,
 		required: true,
 	};
+	const handleDelete =
```

Comment:
Clarify and space line

```suggestion

	const hasDeleletePermission =
```

Reply:
Done!